packages/venia-ui/lib/components/Header/header.js

```diff
@@ -29,13 +29,26 @@ const Header = props => {
     const classes = mergeClasses(defaultClasses, props.classes);
     const rootClass = searchOpen ? classes.open : classes.closed;
     const searchIcon = <Icon src={SearchIcon} />;
-    const suspenseFallback = (
+    const searchBarFallback = (
         <div className={classes.searchFallback}>
             <div className={classes.input}>
                 <div className={classes.loader} />
             </div>
         </div>
     );
+    const searchBar = searchOpen ? (
+        <Suspense fallback={searchBarFallback}>
```

Comment:
1. L40: The `searchBarFallback` is not conditional - it's always the fallback when the `SearchBar` hasn't been downloaded yet.

2. L39: We only ever include `<SearchBar>` in the rendered output when `searchOpen` is `true`. This will actually make it lazy load.